src/disc-golf-events/DiscGolfEventsComponent.js

```diff
@@ -29,46 +31,151 @@ const DiscGolfEventsComponent = () => {
     fetchEvents();
   }, [valueToOrderBy, orderDirection]);
 
-  const fetchEvents = () => {
-    let url = `${config.discGolfServiceUrl}/public/events`;
-    if (valueToOrderBy) {
-      url += "?valueToOrderBy=" + valueToOrderBy;
-      if (orderDirection) {
-        url += "&orderDirection=" + orderDirection.toUpperCase();
+  const fetchEvents = async () => {
+    try {
+      let url = `http://localhost:24001/api/disc-golf-service/public/events`;
```

Comment:
We should stay with config variable `discGolfServiceUrl` as this won't work on some external server.